Flint/src/main/java/flint/CLIAdapter.java

```diff
@@ -0,0 +1,67 @@
+package flint;
+
+import java.util.Collection;
+import java.io.FileNotFoundException;
+
+import flint.FlintConfiguration;
+import flint.LintFailure;
+
+public class CLIAdapter {
+    /**
+     * The function that should be triggered by the command line, this will trigger the
+     * linter to check code style based on the user-defined flint config file.
+     *
+     * @param args - should contain the options and parameters from the command line,
+     *               see Overview for details.
+     **/
+    public static void main(String[] args) throws Exception {
+        throw new Exception("Not Yet Implemented");
+    }
+
+    /**
+     * This function initializes and returns the FlintConfiguration class.
+     *
+     * @param configPath - The path that leads to the FlintConfiguration subclass file.
+     * @return Returns the initialized FlintConfiguration subclass if passed in a valid path.
+     * @throws FileNotFoundException - If configPath did not lead to a file.
+     * @throws IllegalArgumentException - If the passed in configPath is not a FlintConfiguration subclass.
+     * */
+    private static FlintConfiguration configInit(String configPath) throws Exception {
+        throw new Exception("Not Yet Implemented");
+    }
+
+    /**
+     * This function checks if filePath leads to an existing java file.
+     *
+     * @param filePath - The file path to check.
+     * @return Returns the true if the file exists and is a java file. Otherwise false.
+     * */
+    private static boolean isJava(String filePath) throws Exception {

```

Comment:
What exception does this throw and why?
The JavaDoc should state this.

Reply:
I'm throwing exception now just to show that it is not yet implemented, will be removed once the function is done and if no exception is specified in docs